test/html.js

```diff
@@ -73,7 +73,11 @@ describe('html', function() {
     });
 
     let html = fs.readFileSync(__dirname + '/dist/index.html');
```

Comment:
Styling prettier is applying here is kinda hard to read, so to explain:

Check for `\` OR `/` in the path. It will be different on Windows vs Other.